client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -179,36 +158,41 @@ export const LineItemUI = styled( WPLineItem )`
 	justify-content: space-between;
 	font-weight: ${( { theme, total } ) => ( total ? theme.weights.bold : theme.weights.normal) };
 	color: ${( { theme, total } ) => ( total ? theme.colors.textColorDark : theme.colors.textColor) };
-	font-size: ${( { total } ) => ( total ? '1.2em' : '1em') };
+	font-size: ${( { total } ) => ( total ? '1.2em' : '1.1em') };
 	padding: ${( { total, isSummaryVisible, tax, subtotal } ) =>
 		isSummaryVisible || total || subtotal || tax ? '10px 0' : '20px 0'};
 	border-bottom: ${( { theme, total, isSummaryVisible } ) =>
 		isSummaryVisible || total ? 0 : '1px solid ' + theme.colors.borderColorLight};
 	position: relative;
 `;
 
-const LineItemTitleUI = styled.div`
-	flex: 1;
-	word-break: break-word;
-`;
-
-const LineItemPriceUI = styled.span`
-	margin-left: 12px;
+const LineItemMeta = styled.div`
+	color: ${( props ) => props.theme.colors.textColorLight};
+	display: flex;
+	font-size: 13px;
```

Comment:
I've been using 14px for all the other small copy. Can we update it to that here too?